packages/contracts/package.json

```
@@ -22,7 +22,7 @@
     "tsx": "^3.12.2"
   },
   "scripts": {
-    "test": "tsx ./scripts/runner/runner.ts",
+    "test": "forge test",
```

I've noticed issues in the past with conflicting arguments/parameters for shell to pass to yarn -> forge. Are you sure this is stable? Perhaps its better to do: 

```suggestion
    "forge:test": "forge test",
```

---

> Are you sure this is stable? 

Yes, the tests are passing in this PR

> Perhaps its better to do `"forge:test": "forge test"`

We already have `test:slither` and `test:echidna` so perhaps `test:unit` also works